friendshipper/src/lib/components/oobe/WelcomeModal.svelte

```diff
@@ -151,26 +150,23 @@
 	const initiateRepoClone = async () => {
 		errorMessage = '';
 		try {
+			cloning = true;
 			await handleUpdateAppConfig();
 
-			// only clone if this is our first time through
-			if (!currentConfig.repoPath) {
-				cloning = true;
-				await cloneRepo({ url: repoUrl, path: cloneLocation });
+			await cloneRepo({ url: repoUrl, path: cloneLocation });
```

Comment:
Is pulling the repoPath check safe if `cloneLocation` already exists?

Reply:
yep! it's an early out in the backend